### ISSUE TITLE
Ensuring correct ~/.config folder ownership fails on shared hosting

### ISSUE THREAD

**Issue by L3o-pold** [TryGhost/Ghost-CLI#867]
### Summary

On a shared hosting we do not own root user. Ghost will fail during the install because it check that current user has right on the ~/.config folder.

I don't understand why because it don't seems to be required during the install.
I successfully install Ghost on this shared hosting by commenting this step.

### Steps to Reproduce

1. Having a user that doesn't not own the ~/.config folder.
2. Run `ghost install`

### Technical details

This is automatically output by Ghost-CLI if an error occurs, please copy & paste:

* OS: Debian
* Node Version: 8.10.0
* Ghost-CLI Version: 1.9.9
* Environment: production
* Command: `ghost install`

## Log

```bash
uid****@od-***:~/blog3$ ghost install
+ sudo chown -R ****_blog:****_blog /home/clients/*******/.config
? Sudo Password [input is hidden] 
```

I do not own the root password... Could we remove this check or making it optional?

**Comment by vikaspotluri123** [comment]
The folder that Ghost is taking ownership of should be your (the current user) home folder. If the current user isn't able to write to their home folder, Ghost needs to make it writable because other commands depend on configuration that's stored there (e.g `ghost update`). On install, Ghost creates / updates a file in the referenced folder with instance information

You should be able to have ghost skip doing so by ensuring you can write to your home folder.

**Comment by L3o-pold** [comment]
In this shared hosting I have right and read access on the user home folder but a .config folder already exists owned by the root user...
I understand that it may be needed for updating but it seems not used for installing.

**Comment by L3o-pold** [comment]
That's sad because I CAN run Ghost on this shared hosting. But I can't install it if I don't manualy edit the pre-check.
IMO we should warn the user that some features could not work if the Ghost user doesn't have the rights on the `.config` folder but we should not prevent installing for that reason because it actually works!

**Comment by llakssz** [comment]
Similarly, is there any documentation about installing on shared hosting, as in no access to root/sudo?

I could do development installs 1.x but could never get production installs working, I'm not sure if 2.x is different.

**Comment by L3o-pold** [comment]
@llakssz You need to edit the `node_modules/ghost-cli/lib/utils/pre-check.js` file to ignore this requirement.

**Comment by llakssz** [comment]
@L3o-pold Thanks, I manage to _install_ 2.x like this:
`ghost install --db sqlite3 --no-setup-linux-user`
and I skip setting up nginx, systemd, and starting ghost.
I try `ghost start` or `ghost start --no-setup-linux-user --no-setup-systemd`, and I get asked for the sudo password.

The pre-checks.js file you mention is for the install process (I think), but I have completed that stage and am just trying to start ghost.
Thanks

**Comment by L3o-pold** [comment]
Happy that you manage to install it. 
@TryGhost we are one of the most important web hosting provider in Switzerland. It could be nice to make this check optionnal to improve the popularity of Ghost.
If you think it’s not necessary, please close this issue, we will make a fork.

**Comment by acburdine** [comment]
@L3o-pold I can probably add something in that allows you to skip the pre-checks. Thinking about something similar to how you can skip the node-version check with `GHOST_NODE_VERSION_CHECK=false`, might add a `GHOST_CLI_PRE_CHECKS=false` to skip the pre-checks.

**Comment by acburdine** [comment]
@L3o-pold added the environment variable skip in - the fix should be released as part of Ghost-CLI 1.10 once that happens (should hopefully be in the next couple of weeks)
